src/lib/spinel/radio_spinel.hpp

```diff
@@ -353,6 +461,44 @@ template <typename InterfaceType> class RadioSpinel
     otError GetCoexMetrics(otRadioCoexMetrics &aCoexMetrics);
 #endif // OPENTHREAD_CONFIG_PLATFORM_RADIO_COEX_ENABLE
 
+    /**
+     * Get currently active interface.
+     *
+     * @param[out] aIid IID of the interface that owns the radio.
+     *
+     * @retval  OT_ERROR_NONE               Successfully got the property.
+     * @retval  OT_ERROR_RESPONSE_TIMEOUT   Failed due to no response received from the transceiver.
+     * @retval  OT_ERROR_NOT_IMPLEMENTED    Failed due to lack of the support in radio or platform supprts
+     *                                      all interfaces simultaneously
+     *                                      (i.e. no active/inactive interface concept in the platform level)
```

Comment:
Would it be possible to split the error definition? I wonder if the mixed definition could be harder for software higher in the stack to make decisions about.

Reply:
I will add OT_ERROR_INVALID_COMMAND